x-pack/plugins/saved_objects_tagging/public/management/actions/bulk_delete.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { OverlayStart, NotificationsStart } from 'src/core/public';
+import { i18n } from '@kbn/i18n';
+import { ITagInternalClient } from '../../tags';
+import { TagBulkAction } from '../types';
+
+interface GetBulkDeleteActionOptions {
+  overlays: OverlayStart;
+  notifications: NotificationsStart;
+  tagClient: ITagInternalClient;
+  setLoading: (loading: boolean) => void;
+}
+
+export const getBulkDeleteAction = ({
+  overlays,
+  notifications,
+  tagClient,
+  setLoading,
+}: GetBulkDeleteActionOptions): TagBulkAction => {
+  return {
+    id: 'delete',
+    label: i18n.translate('xpack.savedObjectsTagging.management.actions.bulkDelete.label', {
+      defaultMessage: 'Delete',
+    }),
+    icon: 'trash',
+    refreshAfterExecute: true,
+    execute: async (tagIds) => {
+      const confirmed = await overlays.openConfirm(
```

Comment:
@cchaos @MichaelMarcialis What do you two think about somewhere in this modal adding the actual list of tags that will be deleted? (I'm thinking probably at the bottom, a small scrollable list?)

Currently, you'd have to close the modal to review and if the selection spans across pages that could be particularly annoying but I know I always feel a little safer if I can review right before taking a destructive action.

Reply:
If the consensus is to add this, might I suggest creating a separate enhancement issue. That way we'd have some time to iterate on the design as well.

Reply:
Created https://github.com/elastic/kibana/issues/83292

Reply:
Thanks so much for this suggestion, @myasonik! As long as the solution we come up with is scalable and plays nice from within the modal (which your mention of a scrollable list would already indicate), I think this could be a nice addition. I'll add it to my to-do list as a possible enhancement.